src/frontend/ts/state/chessStore.ts

```diff
@@ -1,20 +1,40 @@
 import create from 'zustand'
 import Urbit from '@urbit/http-api'
 import { CHESS } from '../constants/chess'
-import { Update, Ship, GameID, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate, ChallengeSentUpdate, ChallengeReceivedUpdate, PositionUpdate, ResultUpdate, DrawOfferUpdate, DrawDeclinedUpdate, SpecialDrawPreferenceUpdate } from '../types/urbitChess'
+import { Update, Ship, GameID, SAN, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate, ChallengeSentUpdate, ChallengeReceivedUpdate, PositionUpdate, ResultUpdate, DrawOfferUpdate, DrawDeclinedUpdate, SpecialDrawPreferenceUpdate } from '../types/urbitChess'
 import { findFriends } from '../helpers/urbitChess'
 import ChessState from './chessState'
 
 const useChessStore = create<ChessState>((set, get) => ({
   urbit: null,
   displayGame: null,
+  displayMoves: [],
   practiceBoard: '',
   activeGames: new Map(),
   incomingChallenges: new Map(),
   outgoingChallenges: new Map(),
   friends: [],
   setUrbit: (urbit: Urbit) => set({ urbit }),
-  setDisplayGame: (displayGame: ActiveGameInfo | null) => set({ displayGame }),
+  setDisplayGame: (displayGame: ActiveGameInfo | null) => {
+    if (displayGame !== null) {
+      set({ displayGame })
+      // XX: "GameInfo or ActiveGameInfo?" conditional logic
+      //
+      //     might be necessary depending on how we implement
+      //     the "view completed games" feature
+      get().setDisplayMoves(displayGame.info.moves)
+    } else {
+      // XX: get moves list working for practice games
+      //
+      //     will require work in Chessboard.tsx, perhaps
+      //     work on the backend e.g. a new poke to just
+      //     get the PositionUpdate through /updates wire
+      set({ displayGame })
+      get().setDisplayMoves([])
+    }
+  },
+  setDisplayMoves: (displayMoves: Array<SAN>) =>
+    set(state => ({ displayMoves })),
```

Comment:
If `setDisplayMoves` is only used from inside `ChessStore`, there isn't any reason to have it be its own function. May as well just replace both calls to it with `set(state => ({ displayGame.info.moves })` (or `[]`).

PS: Is `state` necessary there, or will `set({ displayGame.info.moves })` work on its own?

Reply:
Display moves might not need to exist anymore, so I'm resolving this conversation

Reply:
Reopening pending resolution to above comment